tests/raftstore/pd.rs

```diff
@@ -505,39 +669,50 @@ impl TestPdClient {
         let region = self.get_region_by_id(region_id).wait().unwrap();
         panic!("region {:?} has peer {:?}", region, peer);
     }
+
     pub fn add_region(&self, region: &metapb::Region) {
         self.cluster.wl().add_region(region)
     }
 
+    pub fn ensure_transfer_leader(&self, region_id: u64, peer: metapb::Peer) {
```

Comment:
Don't need to add `ensure_` if all these operators use `SchedulePolicy::TillSuccess`, like `add_peer` and `remove_peer`.